<p>If <code>testing.py</code> and <code>caller.py</code> are in same folder then the above code should work. But if they are not in same location then it is obvious that you have to provide the location of the file.</p>

<p>I have <code>caller.py</code> in <code>/home/reuben/caller.py</code> and <code>testing.py</code> in <code>/home/reuben/Documents/testing.py</code> . And I have given the full path of testing.py in caller.py. </p>

<pre><code>subprocess.call(['jython', '/home/reuben/Documents/testing.py'])
</code></pre>

<p>And all that is working for me.</p>
